zebra-state/src/service/finalized_state/disk_format.rs

```diff
@@ -292,6 +296,42 @@ impl FromDisk for orchard::tree::NoteCommitmentTree {
     }
 }
 
+#[derive(serde::Serialize, serde::Deserialize)]
+struct HistoryTreeParts {
```

Comment:
Can you open a ticket for adding serialization tests for `HistoryTreeParts` byte test vectors?

I think setting up the tests could get a bit complicated, so I'd prefer to leave it until later. And the risk of a format change is pretty low.